### PR TITLE
Day four

### PR DESCRIPTION
**Pull request by Talaspaev**
I've fixed simple routes mistakes

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -1,31 +1,22 @@
...
-import News from '../Components/News/news';
-import Music from '../Components/Music/music';
-import Settings from '../Components/Settings/settings';
-
-import route from '../route';
+import NavItem from '../Components/Navbar/NavItem/navItem';
 
 import './App.css';
 
 function App() {
```
Comment (Dmitry-White) [inline src/App/App.js]: Consider using either class or arrow function notation for your components.

File: src/App/App.js [hunk 2]
```diff
@@ -1,31 +1,22 @@
...
+  const Items = Object.values(NavItem);
+  const NavItemBlock = Items.map((item) => (<Route key={item.id} path={item.path} component={item.component} />));
```
Comment (Dmitry-White) [inline src/App/App.js]: Consider creating a separate function to hold the map handler and naming it with a "Route" in the name.
E.g const itemRoute = () => {...}

File: src/Components/Navbar/nav.jsx [hunk 3]
```diff
@@ -1,28 +1,18 @@
...
-      <NavLink to={route.music} activeClassName={style.active}>Music</NavLink>
-    </div>
-    <div className={style.item}>
-      <NavLink to={route.settings} activeClassName={style.active}>Settings</NavLink>
-    </div>
-  </nav>
-);
+const NavBar = () => {
+  const Items = Object.values(NavItem);
+  const NavItemBlock = Items.map((item) => (<div key={item.id} className={style.item}><NavLink to={item.path} activeClassName={style.active}>{item.name}</NavLink></div>));
```
Comment (Dmitry-White) [inline src/Components/Navbar/nav.jsx]: Consider creating a separate function to hold the map handler.

Comment (Dmitry-White) [inline src/Components/Navbar/nav.jsx]: Also, consider adding some indentation to your JSX to improve readability.

File: src/App/App.js (hunk 1 above)
Comment (Talaspaev) [reply src/App/App.js]: I changed that

File: src/App/App.js (hunk 2 above)
Comment (Talaspaev) [reply src/App/App.js]: I've added new itemRoute.jsx in my App folder

File: src/Components/Navbar/nav.jsx (hunk 3 above)
Comment (Talaspaev) [reply src/Components/Navbar/nav.jsx]: I made it as a separate function

Comment (Talaspaev) [reply src/Components/Navbar/nav.jsx]: sure I've added some indentation

File: src/App/App.js [hunk 4]
```diff
@@ -1,35 +1,27 @@
...
       <div className="wrapper">
         <Header />
         <NavBar />
         <div className="app-wrapper-contnent">
-          <Route path={route.profile} component={Profile} />
-          <Route path={route.dialogs} component={Dialogs} />
-          <Route path={route.news} component={News} />
-          <Route path={route.music} component={Music} />
-          <Route path={route.settings} component={Settings} />
+          <RouteItem context={Items} />
```
Comment (Dmitry-White) [inline src/App/App.js]: I think, there was a misunderstanding.
All I was suggesting was
1. `const itemRoute = ({id, path, component}) => (<Route key={id} path={path} component={component} />)));`
2. `const NavItemsBlock = Items.map(itemRoute);`

File: src/Components/Navbar/NavItem/navMapFunc.jsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import style from '../nav.module.css';
+
+const NavMapFunc = (props) => {
+  const { context } = props;
+  return (context.map((item) => (<div key={item.id} className={style.item}><NavLink to={item.path} activeClassName={style.active}>{item.name}</NavLink></div>)));
```
Comment (Dmitry-White) [inline src/Components/Navbar/NavItem/navMapFunc.jsx]: Same as above.

**Comment by Dmitry-White** [comment]
Don't forget about my first comment above.

**Comment by Talaspaev** [comment]
I added pre-push and pre-commit next pr

**Pull request merged**